### ISSUE TITLE
Product_data has missing values

### ISSUE THREAD

**Issue by fatemehes75** [Shopify/eng-intern-assessment-data#63]
Product_id is 1 to 30on all other data, except in product_data which is 1 to 16

**Comment by jiminJL** [comment]
Agreed! To specify, product_id 17 and greater are referenced in order_items, reviews, and cart_items. I wonder if it's appropriate if we only test up to product_id 17. 
I hope we get some clarification on this issue since it changes the edge cases we could be testing!

**Comment by RootEncrypter** [comment]
Hey can someone please clarify this or is it clarified yet?
